applications/nrf_desktop/src/modules/Kconfig.ble_qos

```diff
@@ -30,7 +30,8 @@ config DESKTOP_BLE_QOS_STACK_SIZE
 
 config DESKTOP_BLE_QOS_STATS_PRINTOUT_ENABLE
```

Comment:
maybe instead of build assert we should already check the dts nodes at the Kconfig level?
looking at the `zephyr/scripts/kconfig/kconfigfunctions.py` and the NCS overall it might be possible 
```
DT_CHOSEN_NCS_BLE_QOS_UART := ncs,ble-qos-uart
DT_HAS_COMP_ZEPHYR_CDC_ACM_UART := zephyr,cdc-acm-uart
...

depends on $(dt_chosen_enabled,$(DT_CHOSEN_NCS_BLE_QOS_UART))
depends on $(dt_chosen_has_compat,$(DT_CHOSEN_NCS_BLE_QOS_UART),$(DT_HAS_COMP_ZEPHYR_CDC_ACM_UART))
```
Would something like this work?

Reply:
Yes, it seems to work. Thanks!